raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py

```diff
@@ -487,9 +488,11 @@ def test_refund_transfer_next_route():
     assert iteration.new_state is not None
 
     route_cancelled = next(e for e in iteration.events if isinstance(e, EventUnlockFailed))
+    route_failed = next(e for e in iteration.events if isinstance(e, EventRouteFailed))
     new_transfer = next(e for e in iteration.events if isinstance(e, SendLockedTransfer))
```

Comment:
nitpick: It's better to use `search_for_item` here instead of the inlined comprehension.